tests/test_manager.py

```diff
@@ -97,7 +97,7 @@ async def stop(self):
     async def get_block_template(self, address: Optional[str] = None) -> BlockTemplate:
         return self._block_templates[self._current_index]
 
-    async def get_tx_parents(self) -> List[bytes]:
+    async def get_tx_parents(self):
```

Comment:
Is this not used? Seems like a method that should return something, what about `raise NotImplementedError` instead of changing the method signature?

Reply:
Makes sense! This is a method from a class we are extending from the python hathorlib. 

I've rolled back my changes and added the `raise NotImplementedError`, much better. Thanks